### ISSUE TITLE
BIP-47 support for all Bitcoin like networks

### ISSUE THREAD

**Issue by serrq** [horizontalsystems/unstoppable-wallet-android#7537]
https://bitbox.swiss/blog/reusable-payment-codes-exchanging-bitcoin-addresses-privately/

**Comment by retrnull** [comment]
I would like to work on this, but I have a few questions for the maintainers before starting the work:

- Would such a feature be accepted into the codebase?
- I expect to take about a month to implement it. Approximately when would a release including the functionality be made available for download?

I'll start working on it once I get answers.

**Comment by Dianaaiym** [comment]
Closing this issue as it is no longer relevant.
